css/main.css

```diff
@@ -27,7 +27,8 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+#desktop-nav,
```

Comment:
- [ ] Kindly make sure you fix this overlap on your mobile menu. You can always check the Figma template as you are required to stick to it as you are building your portfolio.

![Screenshot from 2022-09-15 10-57-18](https://user-images.githubusercontent.com/97100091/190362180-6a4f4bd4-5bb5-4e8a-ad7a-f780d979f3c2.png)